src/install/z_installer.erl

```diff
@@ -563,11 +576,15 @@ convert_category_hierarchy(C, Database, Schema) ->
             ok
     end.
 
-publication_start_nullable(C, _Database, _Schema) ->
-    {ok, [], []} = pgsql:squery(
-        C,
-        "ALTER TABLE rsc "
-            "ALTER COLUMN publication_start DROP NOT NULL, "
-            "ALTER COLUMN publication_start DROP DEFAULT"
-    ),
-    ok.
+publication_start_nullable(C, Database, Schema) ->
+    case is_column_nullable(C, "rsc", "publication_start", Database, Schema) of
+        true -> ok;
+        false ->
+            {ok, [], []} = pgsql:squery(
+                C,
+                "ALTER TABLE rsc "
+                    "ALTER COLUMN publication_start DROP NOT NULL, "
+                    "ALTER COLUMN publication_start DROP DEFAULT"
+            ),
+            ok
```

Comment:
@mworrell Why did you change this; just to perform fewer `ALTER` operations on the database?

Reply:
yes, otherwise each startup we do those "alter" statements.